examples/planning/python/visualizer.py

```diff
@@ -38,30 +38,41 @@ def cli():
     type=str,
     help="Name of the DMD file which you would like to visualize.",
 )
+@click.option(
+    "--iris",
+    is_flag=True,
+    default=False,
+    help="Show IRIS regions for the given context.",
+)
 @click.option(
     "-f",
     "--force",
     is_flag=True,
     default=False,
     help="Force reload of the visualizer with a new model.",
 )
-def start(id, dmd, force) -> None:
-    print(id, dmd, force)
+def start(id, dmd, force, iris) -> None:
```

Comment:
No documentation? isn't it needed?

Reply:
The documentation is presented by running `python visualizer.py start --help`